### ISSUE TITLE
Edit credit for Design image for redesigned homepage

### ISSUE THREAD

**Issue by JessicaLucindaCheng** [hackforla/website#3294]
### Overview
We need to edit a credit for a new image used in the redesigned version of the homepage so that the [Credits page](https://www.hackforla.org/credits/) is up to date.

### Details
From the [Homepage Redesign Credits in Figma](https://www.figma.com/file/0RRPy1Ph7HafI3qOITg0Mr/Hack-for-LA-Website?node-id=17951%3A122724), here is a screenshot of the credit's information for reference:
  <details>
    <summary>Click here to see the screenshot of the Design credit in Figma</summary>
    <img src="https://user-images.githubusercontent.com/31293603/175382821-b9ffef26-ef23-4465-b5a6-522945afb036.png" alt="">
  </details>

### Action Items
- [x] **Feature Branch:** You will be working off a feature branch called `feature-homepage-launch` and the pull request created from this issue should be merged into that feature branch. Please read and follow the steps in the [How to work off of a feature branch](https://github.com/hackforla/website/wiki/How-to-work-off-of-a-feature-branch) wiki article.
- [x] For `_data/internal/credits/ui-ux.yml`,
  - [x] Change the file name from `ui-ux.yml` to `design2.yml`
  - [x] Change `title: UI/UX` to `title: Design`
  - [x] Change `title-link: https://thenounproject.com/eucalyp/collection/digital-transformation-line/?i=3160558` to `title-link: https://thenounproject.com/search/?q=design&i=3160558`
  - [x] Change `content: icon` to `content-type: image`
  - [x] Change `used-in: Home` to `used-in: Homepage`
  - [x] Change `alt: 'UI/UX Icon'` to `alt: ''`
  - [x] Remove `type: icon`
- [x] Once the pull request associated with this issue is approved and merged, please update and edit epic #3130 by
  - [x] Checking off the dependency for this issue
  - [x] If all dependencies are checked off, please move issue #3130 to the New Issue Approval column and remove the Dependency label
  
### Resources/Instructions
- This issue is tracked in the epic #3130.
- [Credits page](https://www.hackforla.org/credits/)
- [How to work off of a feature branch](https://github.com/hackforla/website/wiki/How-to-work-off-of-a-feature-branch)
- [Credits yml file template for Iconography and Imagery](https://github.com/hackforla/website/wiki/Credits-yml-file-template-for-Iconography-and-Imagery)
- [Homepage Redesign Credits in Figma](https://www.figma.com/file/0RRPy1Ph7HafI3qOITg0Mr/Hack-for-LA-Website?node-id=17951%3A122724)

**Comment by JessicaLucindaCheng** [comment]
This issue would usually have the milestone 01. Compliance. However, it's specifically for the homepage launch, which has not happened yet.

**Comment by clayton1111** [comment]
Availability for this week: All day
My estimated ETA for completing this issue: End of day

**Comment by ExperimentsInHonesty** [comment]
Clayton posted this in slack today
[Clayton Brossia](https://app.slack.com/team/U03LSJHN3PT)
  [10:50 AM](https://hackforla.slack.com/archives/C4UM52W93/p1656525006451329)
Hi everyone. I assigned myself to work on issue [#3294](https://github.com/hackforla/website/issues/3294). I am supposed to be editing the file _data/internal/credits/ui-ux.yml, however the file does not exist. I have been able to find a file called _data/internal/communities/ui-ux.yml , but it is not the same. Any advice, or should I just assign myself to a different issue? (edited)

**Comment by clayton1111** [comment]
![image](https://user-images.githubusercontent.com/107889868/176511410-d47c89d3-f428-4bbf-9818-58e00543520b.png)

**Comment by clayton1111** [comment]
So I figured out that I was in the gh- pages feature branch. However when I try to switch to the feature-homepage-launch feature branch I get this error message:
![image](https://user-images.githubusercontent.com/107889868/176512298-e9f8eb1a-a37a-4155-9be6-5d26e54805fb.png)

**Comment by JessicaLucindaCheng** [comment]
@clayton1111 
> So I figured out that I was in the gh- pages feature branch. However when I try to switch to the feature-homepage-launch feature branch I get this error message: ![image](https://user-images.githubusercontent.com/107889868/176512298-e9f8eb1a-a37a-4155-9be6-5d26e54805fb.png)

Sorry, I wasn't clear in the issue how to work off the feature branch. It looks like you may have figured it out since you have opened a pr. Let me know if you still need help with the blocker you were facing.

**Comment by clayton1111** [comment]
@JessicaLucindaCheng Yes! At first I hadn't switched over to the feature branch (which was my first question). Then when I did I had this error (second question). But I was able to figure out how to fix it and just submitted the PR. Thank you for the support!

**Comment by JessicaLucindaCheng** [comment]
Moving this to Questions/In Review because I am awaiting a response to this question in the associated PR. See: https://github.com/hackforla/website/pull/3325#issuecomment-1175686979

Edit: Nevermind.  I found the other pr that was merged #3326 and linked it to this issue. I'm not sure why the automation didn't work in linking the pr to this issue but I went ahead and manually linked it to this issue.
